integrated_channels/sap_success_factors/models.py

```diff
@@ -63,17 +64,27 @@ class SAPSuccessFactorsEnterpriseCustomerConfiguration(EnterpriseCustomerPluginC
         (USER_TYPE_ADMIN, 'Admin'),
     )
 
-    key = models.CharField(max_length=255, blank=True, verbose_name="Client ID")
-    sapsf_base_url = models.CharField(max_length=255, verbose_name="SAP Base URL")
-    sapsf_company_id = models.CharField(max_length=255, blank=True, verbose_name="SAP Company ID")
-    sapsf_user_id = models.CharField(max_length=255, blank=True, verbose_name="SAP User ID")
-    secret = models.CharField(max_length=255, blank=True, verbose_name="Client Secret")
+    key = models.CharField(
+        max_length=255, verbose_name="Client ID", help_text=_("OAuth client identifier.")
```

Comment:
nit: now that we have a longer list of arguments, we could put each one on one line for better readability.